test/integration/merkle-lockup/ll/clawback/clawback.t.sol

```diff
@@ -8,6 +8,9 @@ import { Errors } from "src/libraries/Errors.sol";
 import { MerkleLockup_Integration_Test } from "../../MerkleLockup.t.sol";
 
 contract Clawback_Integration_Test is MerkleLockup_Integration_Test {
+    uint40 internal firstClaimTime;
```

Comment:
we can declare it in `Defaults`

Reply:
There is no default value for it so I was not sure whats the best place to put it. How about `Integration_Test`?

Reply:
yes it is a default value, the `block.timestamp`

Reply:
Cool. Lets move it to `Defaults` then.